habitat/plan.ps1

```diff
@@ -0,0 +1,31 @@
+$pkg_name="inspec"
+$pkg_origin="chef"
+$pkg_version=$(Get-Content "$PLAN_CONTEXT/../VERSION")
+$pkg_description="InSpec is an open-source testing framework for infrastructure
+  with a human- and machine-readable language for specifying compliance,
+  security and policy requirements."
+$pkg_upstream_url="https://www.inspec.io/"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=('Apache-2.0')
+$pkg_source="https://packages.chef.io/files/stable/inspec/${pkg_version}/windows/2016/inspec-${pkg_version}-1-x64.msi"
+$pkg_shasum="fcdf17b29bc94f2eff1ed59fbb8d224f05df6552e0ff85e1eb71fb4e3b7db24d"
```

Comment:
If the package source is parameterized with the version, the shasum should be as well - this value will only be right once (we hope!)

Reply:
Yep investigating how to do this properly. I think it needs to be put in a file someplace after the omnibus builds the msi just not sure yet how to do that.